hack/terraform/aws/variables.tf

```diff
@@ -78,11 +78,11 @@ variable "aws_vm_os_types_agented" {
       ami_owner       = "099720109477"
     },
     {
-      name            = "ubuntu3"
```

Comment:
would you mind changing this to ubuntu-host1, same as azure tf?

Reply:
Done